altair/utils/core.py

```diff
@@ -668,10 +668,6 @@ def _wrap_in_channel_class(obj, encoding):
             condition = obj["condition"]
         except (KeyError, TypeError):
             pass
```

Comment:
I think we can delete this entire try/except block, `condition` does not seem to be used elsewhere.